spec/models/comment_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject do
+    described_class.new(id: 1, content: '<div>Content</div>', user_id: current_user.id, post_id: current_post.id, reply_id: nil,
+                        created_at: DateTime.now, updated_at: DateTime.now)
+  end
+
+  let(:current_user) { User.create(email: 'jane21@doe.com', password: 'pw123456') }
+  let(:current_post) { Post.create(content: '<div>Content</div>', title: 'Hello Worsld', user_id: current_user.id) }
```

Comment:
If using factories then no need to explicitly pass attributes values, only pass if required.